composer.json

```diff
@@ -4,15 +4,16 @@
 	"require": {
 		"php": "^8.0",
 		"ext-mysqli": "*",
-		"phpstan/phpstan": "^1.2"
+		"phpstan/phpstan": "^1.2",
+		"rector/rector": "^0.12.11"
```

Comment:
remove rector dependency again in https://github.com/staabm/phpstan-dba/issues/51